android-components/components/browser/state/src/test/java/mozilla/components/browser/state/reducer/CopyInternetResourceStateReducerTest.kt

```diff
@@ -0,0 +1,55 @@
+package mozilla.components.browser.state.reducer
+
+import mozilla.components.browser.state.action.CopyInternetResourceAction
+import mozilla.components.browser.state.state.BrowserState
+import mozilla.components.browser.state.state.ContentState
+import mozilla.components.browser.state.state.TabSessionState
+import mozilla.components.browser.state.state.content.ShareInternetResourceState
+import mozilla.components.concept.fetch.Response
+import mozilla.components.support.test.mock
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNotNull
+import org.junit.Assert.assertNull
+import org.junit.Assert.assertTrue
+import org.junit.Test
+
+class CopyInternetResourceStateReducerTest {
+    @Test
+    fun `reduce - AddCopyAction should add the internetResource in the ContentState`() {
```

Comment:
nit: (here and elsewhere): While we still have some tests which don't follow the [`GIVEN-WHEN-THEN` naming scheme](https://martinfowler.com/bliki/GivenWhenThen.html) we are trying to have all new tests following it.

Reply:
Gotcha - I think there's a case for having these follow the existing ShareInternetResourceStateReducerTest? (same as https://github.com/mozilla-mobile/firefox-android/pull/948#discussion_r1121542335)